styles/alert.css

```diff
@@ -0,0 +1,26 @@
+.alert {
+  width: 50%;
+  padding: 1rem;
+  margin: 0.5rem;
+  border-radius: 10px;
+}
+
+.alert-success {
+  background-color: var(--status-online);
+  color: var(--dark-white);
```

Comment:
dont use the color name instead use primary ,secondary,etc

Reply:
Done!!